test: SpannerRetryHelperTest.testExceptionWithRetryInfo failure

In #251 (after moving to GitHub actions), we consistently get this test failure on Windows.

```
[ERROR] Failures: 
[ERROR]   SpannerRetryHelperTest.testExceptionWithRetryInfo:218 expected to be true
[INFO] 
[ERROR] Tests run: 3581, Failures: 1, Errors: 0, Skipped: 0
```

I will merge in #251 to cleanup the README but it would be good if we can investigate this test failure.

--- a/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerRetryHelper.java
+++ b/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerRetryHelper.java
@@ -63,7 +63,7 @@ class SpannerRetryHelper {
   static <T> T runTxWithRetriesOnAborted(Callable<T> callable, RetrySettings retrySettings) {
     try {
       return RetryHelper.runWithRetries(
-          callable, txRetrySettings, new TxRetryAlgorithm<>(), NanoClock.getDefaultClock());
+          callable, retrySettings, new TxRetryAlgorithm<>(), NanoClock.getDefaultClock());
     } catch (RetryHelperException e) {
       if (e.getCause() != null) {
         Throwables.throwIfUnchecked(e.getCause());

--- a/google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerRetryHelperTest.java
+++ b/google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerRetryHelperTest.java
@@ -188,13 +188,18 @@ public class SpannerRetryHelperTest {
 
   @Test
   public void testExceptionWithRetryInfo() {
-    final int RETRY_DELAY_NANOS = 100_000_000;
+    final int RETRY_DELAY_MILLIS = 100;
     Metadata.Key<RetryInfo> key = ProtoUtils.keyForProto(RetryInfo.getDefaultInstance());
     Status status = Status.fromCodeValue(Status.Code.ABORTED.value());
     Metadata trailers = new Metadata();
     RetryInfo retryInfo =
         RetryInfo.newBuilder()
-            .setRetryDelay(Duration.newBuilder().setNanos(RETRY_DELAY_NANOS).build())
+            .setRetryDelay(
+                Duration.newBuilder()
+                    .setNanos(
+                        (int)
+                            TimeUnit.NANOSECONDS.convert(RETRY_DELAY_MILLIS, TimeUnit.MILLISECONDS))
+                    .build())
             .build();
     trailers.put(key, retryInfo);
     final SpannerException e =
@@ -214,8 +219,8 @@ public class SpannerRetryHelperTest {
     // retry info of the exception.
     Stopwatch watch = Stopwatch.createStarted();
     assertThat(SpannerRetryHelper.runTxWithRetriesOnAborted(callable)).isEqualTo(2);
-    long elapsed = watch.elapsed(TimeUnit.NANOSECONDS);
-    assertThat(elapsed >= RETRY_DELAY_NANOS).isTrue();
+    long elapsed = watch.elapsed(TimeUnit.MILLISECONDS);
+    assertThat(elapsed >= RETRY_DELAY_MILLIS).isTrue();
   }
 
   private SpannerException abortedWithRetryInfo(int nanos) {

